cidc_api/models/models.py

```diff
@@ -252,6 +253,14 @@ class Users(CommonColumns):
     role = Column(Enum(*ROLES, name="role"))
     disabled = Column(Boolean, default=False, server_default="false")
 
+    @validates("approval_date")
+    def send_approval_confirmation(self, key, new_approval_date):
+        """Send this user an approval email if their account has just been approved"""
+        if self.approval_date is None and new_approval_date is not None:
```

Comment:
what should happen if `self.approval_date is None`? especially if `new_approval_date is not None`

Reply:
It should send the confirmation email in that case (not sure I understand the question?).

Reply:
`self._approval_date is not None`, sorry